rfc/0054-sdk3-management-apis.md

```diff
@@ -2678,9 +2772,29 @@ interface QueryIndex {
     Optional<String> Condition();
     
     Optional<String> Partition()
+    
+    String BucketName();
+    
+    Optional<String> ScopeName();
```

Comment:
It is kind of inconsistent, because we ask for collection name, but when we retrieve the index back, the user cannot see the collection name back (instead the user should know that it is keyspace). Looks like the implementation details leaking here.

I would propose to define `CollectionName()` method which will return keyspace if the index is scoped, kind of alias of `Keyspace()`, because we are doing same thing here for bucket name, which is keyspace when for collection-less indexes.

Reply:
Done